vuepress/docs/next/tutorials/getting-started/azure-install.md

```diff
@@ -32,141 +30,130 @@ If you're already comfortable setting up an AKS cluster and installing NGINX, th
+6. Enter a name of your choice for the Kubernetes cluster name, e.g. YOUR-CLUSTER
 7. Pick your `Region` if it wasn't automatically selected for you.
-8. In the `Availability zones` dropdown pick __one and only one__ availability zone
+8. In the `Availability zones` dropdown, pick __one and only one__ availability zone
     - Generally, you could pick more than one but it will result in a failure in a quickstart environment. If you chose more than one availability zone you will have to provision storage, manage node affinity, and provide the correct network configuration to ensure your application deploys. We recommend only doing this for production clusters.
-9. Select an [Entando-compatible Kubernetes version](https://www.entando.com/page/en/compatibility-guide), e.g. `1.20.x`
-10. Keep the default `Node size`, e.g. `Standard DS2 v2`
+9. Select an [Entando-compatible Kubernetes version](https://www.entando.com/page/en/compatibility-guide), e.g. 1.21.x
+10. Keep the default `Node size`, e.g. Standard DS2 v2
 11. Keep the `Scale Method` set to `Autoscale` and the `Node count range` set from `1` to `5`
 12. Click `Next: Node Pools` to move to the next tab
-13. Keep the default values here
-    - If you're familiar with AKS you can change these settings as desired
-14. Click `Next: Authentication`
-15. For `Authentication method` select `System-assigned managed identity`
-    - You can also select `Service principal` to have Azure automatically generate one for you. If you use an existing principal, it is up to you to configure it and ensure you have the access you need.
-16. Click `Next: Networking`
-17. Enter a value for DNS name prefix, e.g. `cluster-1-dns`
-18. Click `Review + Create`
-    - Note: There are many other configuration options available for an AKS cluster. Generally, you can change these based on your experience and comfort level with the AKS platform. Entando uses base Kubernetes APIs so as long as you follow the Entando configuration instructions below you can tune your cluster infrastructure to meet your goals.
-19. Select `Create`
-20. Wait for your cluster to initialize. This may take a few minutes.
-
-Note: A different storage class can be configured by following [these instructions](./gke-install.md#appendix-a-configuring-clustered-storage).
+13. For `Node Pools`, keep the default values
+    - If you're familiar with AKS, you can change these settings as desired
+    - Click `Next: Access`
+14. For `Access`, keep the default values, e.g. System-assigned managed identity
+15. Click `Next: Networking`
+16. Keep the default settings
+17. Click `Review + Create`
+    - Note: There are many other configuration options available for an AKS cluster. Generally, you can change these based on your experience and comfort level with the platform. Entando uses base Kubernetes APIs so as long as you follow the Entando configuration instructions below you can tune your cluster infrastructure to meet your goals.
```

Comment:
added commas >> Entando uses base Kubernetes APIs, so as long as you follow the Entando configuration instructions below, you can tune...